app/decorators/case/sar/offender_decorator.rb

```diff
@@ -27,4 +27,13 @@ def rejected_reasons_descriptions
       "#{Case::SAR::Offender::REJECTED_REASONS[reason]}#{suffix}"
     }.compact.join("<br>")
   end
+
+  def highlight_flag
+    if object.flag_as_high_profile?
+      '<div class="offender_sar-profile_flag"><span class="visually-hidden">' \
+        'This is a </span>High priority<span class="visually-hidden"> case</span></div>'
```

Comment:
This is the wrong text value. According to the ticket it should be "High Profile"